package.json

```diff
@@ -10,10 +10,10 @@
   },
   "dependencies": {
     "@tailwindcss/postcss7-compat": "^2.0.3",
+    "@videsk/humanize-date": "^1.0.0",
     "apexcharts": "^3.25.0",
     "autoprefixer": "^9.8.6",
     "core-js": "^3.9.1",
-    "moment": "^2.29.1",
```

Comment:
Don't use moment, is deprecated and unnecessary, ES6 has a natively dates parser!